core/libraries/messages/data_class/EE_Messages_Preview_incoming_data.class.php

```diff
@@ -129,23 +137,24 @@ private function _setup_attendees_events()
                 foreach ($reldatetime as $datetime) {
                     if ($datetime instanceof EE_Datetime && ! isset($dtts[ $datetime->ID() ])) {
                         $this->_events[ $id ]['dtt_objs'][ $datetime->ID() ] = $datetime;
-                        $dtts[ $datetime->ID() ]['datetime']               = $datetime;
-                        $dtts[ $datetime->ID() ]['tkt_objs'][]             = $ticket;
-                        $dtts[ $datetime->ID() ]['evt_objs'][]             = $event;
-                        $dttcache[ $datetime->ID() ]                       = $datetime;
+                        $dtts[ $datetime->ID() ]['datetime']                 = $datetime;
+                        $dtts[ $datetime->ID() ]['tkt_objs'][]               = $ticket;
+                        $dtts[ $datetime->ID() ]['evt_objs'][]               = $event;
+                        $dttcache[ $datetime->ID() ]                         = $datetime;
                     }
                 }
             }
-            
+
             $this->_events[ $id ]['total_attendees'] = count($attendees);
             $this->_events[ $id ]['att_objs']        = $attendees;
-            
+
             // let's also setup the dummy attendees property!
             foreach ($attendees as $att_key => $attendee) {
                 if (! $attendee instanceof EE_Attendee) {
                     continue;
                 }
-                $this->_attendees[ $att_key ]['line_ref'][] = $id;  // so later it can be determined what events this attendee registered for!
+                $this->_attendees[ $att_key ]['line_ref'][] =
+                    $id;  // so later it can be determined what events this attendee registered for!
```

Comment:
```suggestion
                // so later it can be determined what events this attendee registered for!
                $this->_attendees[ $att_key ]['line_ref'][] = $id;
```